docs/current/cookbook.md

```diff
@@ -655,6 +655,33 @@ The following code listing demonstrates how to inject a file in a container as a
 
 [Learn more](./guides/723462-use-secrets.md)
 
+### Use secret in Dockerfile build
+
+The following code listing demonstrates how to inject a secret into a Dockerfile build. The secret is automatically mounted in the build container at `/run/secrets/SECRET-ID`.
+
```

Comment:
It would also make sens to show the Dockerfile here, I know that the `learn more` section is here for that but the reader shall have access to an example at first sight IMO.

Reply:
Added